aptos-move/framework/core/sources/Account.move

```diff
@@ -125,6 +125,7 @@ module CoreFramework::Account {
         authentication_key_prefix: vector<u8>,
         _witness: &T,
     ): (signer, vector<u8>) {
+        authentication_key_prefix = Vector::empty<u8>();
```

Comment:
this is too hacky, we should cleanup the interface

Reply:
Yes, this will be in the follow up issue https://github.com/aptos-labs/aptos-core/issues/246